[git/en] add git blame (#4474)
adambard/learnxinyminutes-docs: git.html.markdown

$ git rm /pather/to/the/file/HelloWorld.c
```

### blame

> Examine specific points the code's history and find out who the last author was on that modified line

Reviewer: ```suggestion
Examine specific parts of the code's history and find out who was the last author to modify that line
```
